src/main/java/Duke.java

```diff
@@ -1,77 +1,176 @@
+                + "[D]" + "[ " + "] " + deadlineDescription + "(by: " + deadlineBy + ")" + System.lineSeparator()
+                + "You now have " + tasksCounter + " tasks in the list!" + System.lineSeparator()
+                + "____________________________________________________________" + System.lineSeparator();
+        System.out.println(addedMessage);
+    }
+
+    public static void printAddedEvent(String eventDescription, String eventAt) {
+        String addedMessage = "____________________________________________________________" + System.lineSeparator()
+                + "Alright! I've successfully added this task:"  + System.lineSeparator()
+                + "[E]" + "[ " + "] " + eventDescription + "(at: " + eventAt + ")" + System.lineSeparator()
+                + "You now have " + tasksCounter + " tasks in the list!" + System.lineSeparator()
+                + "____________________________________________________________" + System.lineSeparator();
+        System.out.println(addedMessage);
+    }
+
+    public static void printList() {
+        System.out.println("___________________________________________________________" + System.lineSeparator()
+                + "As requested, here are the tasks in your list:");
+        if (tasksCounter == 0) {
+            System.out.println("There are no tasks recorded!");
+        }
+        for (int i = 0; i < tasksCounter; i++) {
+            String taskType = tasks[i].getType();
+            String byOrAt = "";
+
+            if (taskType.equals("D")) {
+                byOrAt = "by: ";
+            }
+            else if (taskType.equals("E")) {
+                byOrAt = "at: ";
+            }
+
+            if (byOrAt.equals("")) {
+                System.out.println(i + 1 + ". "
+                        + "[" + taskType + "]"
+                        + "[" + tasks[i].getStatusIcon() + "] "
+                        + tasks[i].description);
+            }
+            else {
+                System.out.println(i + 1 + ". "
+                        + "[" + taskType + "]"
+                        + "[" + tasks[i].getStatusIcon() + "] "
+                        + tasks[i].description + "(" + byOrAt + tasks[i].getWhen() + ")");
+            }
+        }
+        System.out.println("____________________________________________________________");
     }
 
     public static void printExit() {
-        String exitMessage = "____________________________________________________________\n"
-                + "Thanks for talking with me, see you soon!\n"
-                + "____________________________________________________________\n";
+        String exitMessage = "____________________________________________________________" + System.lineSeparator()
+                + "Thanks for talking with me, see you soon!" + System.lineSeparator()
+                + "____________________________________________________________";
         System.out.println(exitMessage);
```

Comment:
Constants can be defined as static final with name of EXIT_MESSAGE.

Reply:
Did not realize this earlier until I looked through everyone's PR as well. Thanks for the suggestion!